backupstoragelocation.md

```diff
@@ -81,4 +81,11 @@ spec:
     #
     # Optional (defaults to "false").
     insecureSkipTLSVerify: "true"
+
+    # Set this to "true" if you do not want to load the credentials file as a [shared config file](https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html).
```

Comment:
This should be set to "false" if you you do not want to use the shared config format.

Reply:
Sorry that was a silly mistake :) should be updated now.